<ui version="4.0" >
 <class>frmClient</class>
 <widget class="QDialog" name="frmClient" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Drink Client</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="logos.qrc" >
    <normaloff>:/Icons/images/soda_32.png</normaloff>:/Icons/images/soda_32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDrink" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>458</width>
        <height>345</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Drink</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10" >
       <item>
        <widget class="QScrollArea" name="scrollArea" >
         <property name="widgetResizable" >
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>436</width>
            <height>286</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4" >
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" native="1" name="wgtDrink" >
             <layout class="QVBoxLayout" name="verticalLayout_7" />
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Delay:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spnDrinkDelay" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLittleDrink" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>458</width>
        <height>345</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Little Drink</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QScrollArea" name="scrollArea_2" >
         <property name="widgetResizable" >
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>436</width>
            <height>286</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5" >
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" native="1" name="wgtLittleDrink" >
             <layout class="QVBoxLayout" name="verticalLayout_8" />
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Delay:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spnLittleDrinkDelay" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSnack" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>458</width>
        <height>345</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Snack</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QScrollArea" name="scrollArea_3" >
         <property name="widgetResizable" >
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>436</width>
            <height>286</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6" >
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" native="1" name="wgtSnack" >
             <layout class="QVBoxLayout" name="verticalLayout_9" />
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Delay:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spnSnackDelay" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="lblCredits" >
       <property name="text" >
        <string>You have &lt;b>?&lt;/b> credits.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="logos.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>frmClient</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>413</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel" >
     <x>474</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
